spikeextractors/extractors/shybridextractors/shybridextractors.py

```diff
@@ -0,0 +1,174 @@
+
+    def get_sampling_frequency(self):
+        return self._bindatext.get_sampling_frequency()
+
+    def get_traces(self, channel_ids=None, start_frame=None, end_frame=None):
+        return self._bindatext.get_traces(channel_ids=channel_ids,
+                                          start_frame=start_frame,
+                                          end_frame=end_frame)
+
+    @staticmethod
+    def write_recording(recording, save_path, initial_sorting_fn, dtype='float32'):
+        """ Convert and save the recording extractor to SHYBRID format
+
+        parameters
+        ----------
+        recording: RecordingExtractor
+            The recording extractor to be converted and saved
+        save_path: str
+            Full path to desired target folder
+        initial_sorting_fn: str
+            Full path to the initial sorting csv file (can also be generated
+            using write_sorting static method from the SHYBRIDSortingExtractor)
+        dtype: dtype
+            Type of the saved data. Default float32.
+        """
+        RECORDING_NAME = 'recording.bin'
+        PROBE_NAME = 'probe.prb'
+        PARAMETERS_NAME = 'recording.yml'
+
+        # location information has to be present in order for shybrid to
+        # be able to operate on the recording
+        if 'location' not in recording.get_shared_channel_property_names():
+            raise GeometryNotLoadedError("Channel locations were not found")
+
+        # write recording
+        recording_fn = os.path.join(save_path, RECORDING_NAME)
+        BinDatRecordingExtractor.write_recording(recording, recording_fn,
+                                                 time_axis=0, dtype=dtype)
+
+        # write probe file
+        probe_fn = os.path.join(save_path, PROBE_NAME)
+        save_to_probe_file(recording, probe_fn, format='spyking_circus')
+
+        # create parameters file
+        source_dir, _ = os.path.split(__file__)
+        parameters_template_fn = os.path.join(source_dir, 'parameters_template')
+
+        with open(parameters_template_fn, 'r') as fp:
+            parameters_template = fp.read()
+
+        parameters = parameters_template.format(initial_sorting_fn=initial_sorting_fn,
+                                                data_type=dtype,
+                                                sampling_frequency=str(recording.get_sampling_frequency()),
+                                                byte_ordering='F',
+                                                probe_fn=probe_fn)
+
+        # write parameters file
+        parameters_fn = os.path.join(save_path, PARAMETERS_NAME)
+        with open(parameters_fn, 'w') as fp:
+            fp.write(parameters)
+
+
+class SHYBRIDSortingExtractor(SortingExtractor):
+    extractor_name = 'SHYBRIDSortingExtractor'
+    installed = HAVE_SBEX
+    extractor_gui_params = [
+        {'name': 'gt_fn', 'type': 'file', 'title': "Full path to hybrid ground truth labels (.csv)"},
+    ]
+    is_writable = False
```

Comment:
this should be `True` as you have a write function. Can you add it to the RecordingExtractor as well?